src/py42/_internal/archive_access.py

```diff
@@ -160,13 +160,17 @@ def is_job_complete(self, job_id):
         return self._get_completion_status(response)
 
     def _start_restore(self, file_selection):
+        zip_result = (
```

Comment:
I just want to my example in the docs to always be True, that is why I thought it might safer to just to explicitly request a zipped result when restoring directories.